README.md

```diff
@@ -132,6 +132,75 @@ is not supported anymore. Exporting `$ROLENAME_provider_os_default` allows the u
 in Ansible) and thus get a consistent setting for all the systems of the given OS version without
 having to decide what the actual value is - the decision is delegated to the role.)
 
+Coding Conventions
+===========
+## Patterns and Caveats
+
+* Jinja2 templates are named with `.j2` extension.  
+* Ansible variables use lazy evaluation. [more_info](https://github.com/ansible/ansible/issues/10374)
+* All tags should be namespaced/prefixed with the role name. 
+* Use the `command` module instead of the `shell` module when use of either is required. In all other cases use a dedicated module, if it exists. If not, see the [section](#check-mode-and-idempotency-issues) about idempotency and check mode and make sure that you support them properly (your task will likely need conditionals such as `changed_when:` and maybe `check_mode:` ). Anytime `command` or `shell` modules are used; a comment in the code with justificiation would help with future maintenance.
```

Comment:
> "...when use of either is required." Is your intention to make one a preference over the other? Did you mean to say "Use the `command` module instead of the `shell` module unless there is a specific need for `shell` features" or something similar?

Our draft says: "Use preferably the command module instead of the shell module. Even better, use a dedicated module, if it exists. " Would that be clear?

Reply:
That would be clearer. I can see what the original phrasing is getting at, it's just awkward, because in this bullet there's not discussion of other modules. So the "when use of either is required" threw my mind off at first.

Reply:
What do you mean by "original phrasing"? The one in the draft, or the one originally in this PR? (I suppose the latter, but for me "the original" is the former.)

Reply:
I mean in the PR.